deployments/testent.md

```diff
@@ -0,0 +1,33 @@
+## Testnet Deployments
```

Comment:
Little typo in the filename. Should be named `testnet.md` instead of `testent.md`.